From c9a952890e96e50ec8d004ee7681aa932bd24db8 Mon Sep 17 00:00:00 2001
From: Louis Bouchard <louis.bouchard@canonical.com>
Date: Wed, 27 Aug 2014 15:00:57 +0200
Subject: collect-cloud-init-log

Add collection of /var/log/cloud-init.log
 and /var/log/cloud-init-output.log if they exist.

Author: Louis Bouchard <louis.bouchard@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1301819
Origin: upstream, https://github.com/sosreport/sos/commit/36efd32ab01df2c076c3271543d674a515094375
---
 sos/plugins/logs.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sos/plugins/logs.py b/sos/plugins/logs.py
index 7e7545f..323f639 100644
--- a/sos/plugins/logs.py
+++ b/sos/plugins/logs.py
@@ -36,6 +36,7 @@ class Logs(Plugin):
 
         self.limit = self.get_option("logsize")
         self.add_copy_spec_limit("/var/log/boot*", sizelimit = self.limit)
+        self.add_copy_spec_limit("/var/log/cloud-init*", sizelimit = self.limit)
 
         if self.get_option('all_logs'):
             logs = self.do_regex_find_all("^\S+\s+(-?\/.*$)\s+",
